lib/dom.js

```diff
@@ -293,6 +293,7 @@ declare class Document extends Node {
     createElement(tagName: 'canvas'): HTMLCanvasElement;
     createElement(tagName: 'div'): HTMLDivElement;
     createElement(tagName: 'form'): HTMLFormElement;
+    createElement(tagName: 'iframe'): HTMLIFrameElement;
```

Comment:
no, the test failures are related because this line pushed down all of the line numbers in the expected error messages. `cp tests/dom/dom.{out,exp}` to update the expected output.